cmd/bad-key-revoker/main.go

```diff
@@ -357,10 +362,16 @@ func main() {
 			// keyHashToSerial table at once
 			FindCertificatesBatchSize int
 
-			// Interval specifies how long bad-key-revoker should sleep between attempting to find
-			// blockedKeys rows to process when there is no work to do
+			// Interval specifies the minimum duration bad-key-revoker
+			// should sleep between attempting to find blockedKeys rows to
+			// process when there is an error or no work to do.
 			Interval cmd.ConfigDuration
 
+			// BackoffDurationMax specifies a maximum amount of time the
+			// backoff algorithm will wait before retrying in the event of
+			// error or no work to do.
+			BackoffDurationMax cmd.ConfigDuration
```

Comment:
Definitely a nit, and feel free to disagree, but I'd call this `BackoffIntervalMax` just to match `Interval` above. Then both fields in the config will match, and both "intervals" can get converted to "durations" as the config is turned into a badKeyRevoker struct.

Reply:
Done! I think `Interval` is probably an unnecessary config flag after this change and we could probably remove it in the future.